taigit/src/libraries/Taiga.ts

```diff
@@ -11,44 +17,77 @@ taiga_login(username : string, password : string) : Promise<boolean> {
     return (response.status == 200);
 }
 
-// This call returns key project information
+/**
+ * @summary Get the project information for a slug
+ * @param slug the url name of the project
+ * @returns object of project information
+ * {
+ *      id : number,          // Project id
+ *      name : string,        // Project name
+ *      created_date : number // Date project was created
+ *      members: Object[]     // Array containing role (number), string for role_name, full_name, username
+ * }
+ */
 export async function
 project_info(slug : string) : Promise<Object> {
     let data = await axios.get("https://api.taiga.io/api/v1/projects/by_slug?slug=" + slug)
-    let info : {id: number, name: string, slug: string, created_date: Date} =
-        {id: data.data.id, name: data.data.name, slug: data.data.slug, created_date: data.data.created_date};
+    let info : {id: number, name: string, created_date: Date, members: Object[]} =
+        {id: data.data.id, name: data.data.name, created_date: data.data.created_date, members: data.data.members};
```

Comment:
Maybe it would more useful to define an object for this elsewhere instead of doing it inline. it would be cleaner and probably more descriptive